### DISCUSSION TITLE
Is Meson supposed to skip test on cross build?

### DISCUSSION THREAD

**Discussion by dreamer-coding** [mesonbuild/meson#13326] [Q&A] [answered]
Is Meson supposed to skip test on cross-build? I'm curious because I just added a nice collection of cross-files to my test framework to validate it can run test on these bedrock host systems, compilation all good but I don't know if skipping the test is part of the show.

**Comment by nils-werner** [comment]
I don't understand what you're asking... do you want Meson do skip tests, or is it skipping them against your will?

Did you set up a `exe_wrapper` in your cross files?

**Reply by dreamer-coding** [reply]
It's skipping them against my will. I kept on trying to run the test command

**Reply by nils-werner** [reply]
Did you set up `exe_wrapper` in your cross files?

**Reply by dreamer-coding** [reply]
No.

**Reply by nils-werner** [reply]
You have to, otherwise you won't be able to run cross compiled binaries.

**Comment by dreamer-coding** [comment]
how can I do that?

**Reply by nils-werner** [reply, marked as answer]
https://mesonbuild.com/Cross-compilation.html#binaries
